packages/instant-apps-components/src/components/instant-apps-landing-page/instant-apps-landing-page.test.ts

```diff
@@ -0,0 +1,82 @@
+import { expect, test, describe } from 'vitest';
+import '../../../dist/components/instant-apps-landing-page.js';
+const componentConfig = {
+  enableSignIn: 'true',
+  titleText: 'Vitest Instant App',
+  subtitleText: 'Hi this is only for testing purposes',
+  descriptionText: "Áádóó naashnishgo, náásht'įįh łeets'óózh jiní. Diné be'elyá bee iináanii át'éego, shił yá'",
+  entryButtonText: 'Click me!',
+  iconImageScale: 's',
+  iconImage: 'https://www.esri.com/content/dam/esrisites/en-us/arcgis/products/arcgis-instant-apps/assets/instant-apps-banner-fg.png',
+};
+
+describe('uninitialized enableSignIn Prop', async () => {
```

Comment:
These tests can be wrapped in `describe('Landing page'`. Create the landing page element and set the text attribute values once.